client/state/selectors/get-sites-logmein-allowed-urls.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Internal dependencies
+ */
+import getSites from 'calypso/state/selectors/get-sites';
+
+/**
+ * Returns a list of users domains that need pushing through logmein
+ *
+ * @param  {object}  state  Global state tree
+ * @returns {?string}       URLs of user's simple sites with mapped domains
+ */
+export default function getSitesLogmeinAllowedUrls( state ) {
+	const result = getSites( state )
+		.map( ( site ) =>
+			! site.is_vip &&
+			! site.jetpack &&
+			! site.options.is_automated_transfer &&
+			! site.options.is_domain_only &&
+			! site.options.is_redirect &&
+			! site.options.is_wpcom_atomic &&
+			! site.options.is_wpcom_store &&
+			! site.options.is_wpforteams_site &&
+			site.options.is_mapped_domain
+				? site.URL
+				: false
+		)
+		.filter( Boolean );
+	return result;
```

Comment:
I'm not sure if this is the correct list of restrictions.. do we have an "is simple site" method by chance?

Reply:
I don't know the right answer to this, but here's what I found. 
We have [this selector](https://github.com/Automattic/wp-calypso/blob/b703449c2b079916da70356acd66ca823760cebf/client/state/selectors/is-site-wpcom.js) that returns if a site is a WPCOM site(atomic or simple), as described the JS Doc:

> Whether the site is a WPCOM site ( Atomic or Simple )

Using the logic in that selector, the following should return simple sites:

```
! isJetpackSite( state, siteId ) || ! isAtomicSite( state, siteId )
```

Would this be the right check for WPCOM simple sites?